src/features/student/views/StudentToolsTracesView/components/TraceAuthenticDeclarationToggleFormField/TraceAuthenticDeclarationToggleFormField.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+import type { CreateTraceForm, UpdateTraceForm } from '@/features/student/types'
+import { TraceAuthenticDeclarationToggle } from '@/features/student/components/toggles'
+
+interface TraceAuthenticDeclarationToggleFormFieldProps {
+  form: CreateTraceForm | UpdateTraceForm
+}
+
+const { form } = defineProps<TraceAuthenticDeclarationToggleFormFieldProps>()
+
+const isAuthenticField = form.useField({ name: 'isAuthentic' })
+const authenticErrors = computed(() => isAuthenticField.state.value.meta.errors)
+</script>
+
+<template>
+  <div>
+    <form.Field name="isAuthentic">
+      <template #default="{ field }">
+        <TraceAuthenticDeclarationToggle
+          v-bind="$attrs"
+          id="isAuthentic"
+          name="isAuthentic"
+          :model-value="field.state.value"
+          @update:model-value="(value) => field.handleChange(value)"
+        />
+      </template>
+    </form.Field>
+    <div
+      v-if="authenticErrors.length > 0"
+      class="trace-form__authentic-error"
+    >
+      {{ authenticErrors.join(', ') }}
```

Comment:
```suggestion
      <span class="caption-regular">{{ authenticErrors.join(', ') }}</span>
```
Toujours les textes dans un span avec une classe de typographie reconnue et associer la couleur dans le style ;)